tests/storage/datavolume.go

```diff
@@ -460,6 +460,9 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 					realRegistryName = strings.Split(flags.KubeVirtUtilityRepoPrefix, ":")[0]
 					realRegistryPort = strings.Split(flags.KubeVirtUtilityRepoPrefix, ":")[1]
 				}
+				if realRegistryPort == "" {
+					Skip("Skip when no port, CDI will always try to reach dockerhub/fakeregistry instead of just fakeregistry")
+				}
```

Comment:
I prefer not to skip tests given the opportunity. It seems like below, when defining fakeRegisterWithPort, we can supply a default port 5000 if it is not specified. What do you think about this alternative?

Reply:
Tried, failing on wildcards (quay.io is ok, add port, not anymore)